.gitlab-ci.yml

```diff
@@ -42,6 +42,9 @@ parthenon-cuda-short:
       -DKokkos_ENABLE_CUDA_LAMBDA=True
       -DCMAKE_CXX_COMPILER=${PWD}/../external/Kokkos/bin/nvcc_wrapper
       ../ && make -j${J} && make -j${J} test
+  artifacts:
+    paths:
+      - /builds/pgrete/parthenon/build-cuda/CMakeFiles/CMakeOutput.log
```

Comment:
Is there a way to not use absolute paths here? `/builds/pgrete/parthenon/build-cuda/` could change...

Reply:
Yes you're right. I'll figure out what the right path is.